frontend/src/pages/AccountSend.vue

```diff
@@ -144,8 +195,18 @@
       </div>
 
       <!-- Send button -->
+      <!-- Pop modal and send in that -->
```

Comment:
A bit confused by these two comment lines—Is one outdated? Also not sure what "Pop modal and send in that" means 😅

Reply:
Lol, oops must have accidentally left a note behind